docs/iris/src/developers_guide/release.rst

```diff
@@ -1,75 +1,172 @@
+`conda forge`_.  
 
-Once a release is cut, the SciTools conda-recipes-scitools recipe for Iris shall be updated to build the latest release of Iris and push this artefact to anaconda.org.  The build and push is all automated as part of the merge process.
+.. _Iris feedstock: https://github.com/conda-forge/iris-feedstock/tree/master/recipe
+.. _conda forge: https://anaconda.org/conda-forge/iris
 
 Merge back
-==========
+----------
+
+After the release is cut, the changes shall be merged back onto the scitools
+master.
 
-After the release is cut, the changes shall be merged back onto the scitools master.
+To achieve this, first cut a local branch from the release branch,
+:literal:`{release}.x`.  Next add a commit changing the release string to match
+the release string on scitools/master.  This branch can now be proposed as a
+pull request to master.  This work flow ensures that the commit identifiers are
+consistent between the :literal:`.x` branch and :literal:`master`.
 
-To achieve this, first cut a local branch from the release branch, :literal:`{release}.x`.  Next add a commit changing the release string to match the release string on scitools/master.  
-This branch can now be proposed as a pull request to master.  This work flow ensures that the commit identifiers are consistent between the :literal:`.x` branch and :literal:`master`.
 
 Point releases
-==============
+--------------
 
-Bug fixes may be implemented and targeted as the :literal:`.x` branch.  These should lead to a new point release, another tag.
-For example, a fix for a problem with 1.9.0 will be merged into 1.9.x, and then released by tagging 1.9.1.
+Bug fixes may be implemented and targeted as the :literal:`.x` branch.  These
+should lead to a new point release, another tag.  For example, a fix for a
+problem with 1.9.0 will be merged into 1.9.x, and then released by tagging
+1.9.1.
 
 New features shall not be included in a point release, these are for bug fixes.
 
-A point release does not require a release candidate, but the rest of the release process is to be followed, including the merge back of changes into :literal:`master`.  
+A point release does not require a release candidate, but the rest of the
+release process is to be followed, including the merge back of changes into
+:literal:`master`.  
+
+
+.. _iris_development_releases_steps:
+
+Maintainer steps
+----------------
+
+These steps assume a release for ``v1.9`` is to be created
+
+Release steps
+~~~~~~~~~~~~~
+
+#. Create a branch on the main repo, not in a forked repo, for the release
+   candidate or release
+#. Update the what's new for the release:  
+
+    * Copy ``docs/iris/src/whatsnew/latest.rst`` to a file named
+      ``v1.9.rst``
+    * Delete the ``docs/iris/src/whatsnew/latest.rst`` file so it will not
+      cause an issue in the build
+    * In ``v1.9.rst`` update the page title (first line of the file) to show
+      the date and version in the format of ``v1.9 (DD MMM YYYY)``
```

Comment:
@tkknight Perhaps give an example here for the date format to avoid confusion e.g., `01 Jan 1970`